docs/source/install/rhel7.rst

```diff
@@ -34,6 +39,9 @@ you may want to tweak them according to your security practices.
     # SELINUX management tools, not available for some minimal installations
     sudo yum install -y policycoreutils-python
 
+    # SELINUX python3 rules
+    sudo yum install -y libselinux-python3
+
     # Allow network access for nginx
     sudo setsebool -P httpd_can_network_connect 1
```

Comment:
Following the discussion in https://github.com/StackStorm/ansible-st2/pull/278 we discovered that `libselinux` is the requirement for specific Ansible module, but not for the st2 installation itself.

I also don't see it between the requirements for EL8 installation:
https://github.com/StackStorm/st2-packages/blob/b3e6674de987198b88f0fe3a3caf332e5037059c/scripts/st2bootstrap-el8.sh#L478-L485

With that, looks like we can remove this dependency, not sure though where it came from as `setsebool` tool required `policycoreutils-python` package only.

Reply:
Snap - was just about to remove it!